tests/multi_objective_tests/test_study.py

```diff
@@ -171,21 +171,20 @@ def test_log_completed_trial_skip_storage_access() -> None:
 
     study = optuna.multi_objective.create_study(["minimize", "maximize"])
 
-    new_trial_id = study._study._storage.create_new_trial(study._study._study_id)
-    trial = optuna.Trial(study._study, new_trial_id)
+    study._study._storage.create_new_trial(study._study._study_id)
+    frozen_trial = study.trials[0]
     storage = study._storage
 
     with patch.object(storage, "get_trial", wraps=storage.get_trial) as mock_object:
-        optuna.multi_objective.study._log_completed_trial(study, trial, [1.0])
-        # Trial.params and MultiObjectiveTrial._get_values access storage.
-        assert mock_object.call_count == 2
+        optuna.multi_objective.study._log_completed_trial(study, frozen_trial)
+        assert mock_object.call_count == 0
```

Comment:
https://github.com/optuna/optuna/pull/3144/files#diff-07b9f79024da1a09ad276d497bf17b34debf8821f87907777a5942935d1816a8R483-R492

`_log_completed_trial` doesn't access to the storage any more.

Reply:
`test_log_completed_trial_skip_storage_access` can be removed in my opinion. It makes a lot less sense now without contextual background to readers. Not a strong opinion, though.